Makefile

```diff
@@ -23,25 +23,33 @@ clean:
 
 makemessages: /tmp/template.pot
 
-translations: ./example/translations.json
+translations: ./$(OUTPUT_DIR)/translations.json
 
 # Create a main .pot template, then generate .po files for each available language.
+# Thanx to Systematic: https://github.com/Polyconseil/systematic/blob/866d5a/mk/main.mk#L167-L183
 /tmp/template.pot: $(GETTEXT_HTML_SOURCES)
+# `dir` is a Makefile built-in expansion function which extracts the directory-part of `$@`.
+# `$@` is a Makefile automatic variable: the file name of the target of the rule.
+# => `mkdir -p /tmp/`
 	mkdir -p $(dir $@)
 	which gettext-extract
+# Extract gettext strings from templates files and create a POT dictionary template.
 	gettext-extract --quiet --output $@ $(GETTEXT_HTML_SOURCES)
-	xgettext --language=JavaScript --keyword=i18n --keyword=npgettext:1c,2,3 \
-		 --from-code=utf-8 --sort-output --join-existing --no-wrap \
+# Extract gettext strings from JavaScript files.
+	xgettext --language=JavaScript --keyword=npgettext:1c,2,3 \
+		--from-code=utf-8 --sort-output --join-existing --no-wrap \
+		--package-name=$(shell node -e "console.log(require('./package.json').name);") \
 		--package-version=$(shell node -e "console.log(require('./package.json').version);") \
 		--output $@ $(GETTEXT_JS_SOURCES)
+# Generate .po files for each available language.
 	@for lang in $(LOCALES); do \
 		export PO_FILE=$(OUTPUT_DIR)/locale/$$lang/LC_MESSAGES/app.po; \
-		echo "msgmerge --sort-output --update $$PO_FILE $@"; \
+		echo "msgmerge --update $$PO_FILE $@"; \
 		mkdir -p $$(dirname $$PO_FILE); \
-		[ -f $$PO_FILE ] && msgmerge --lang=$$lang --sort-output --update $$PO_FILE $@ || msginit --no-translator --locale=$$lang --input=$@ -o $$PO_FILE; \
+		[ -f $$PO_FILE ] && msgmerge --lang=$$lang --update $$PO_FILE $@ || msginit --no-translator --locale=$$lang --input=$@ --output-file=$$PO_FILE; \
```

Comment:
Hm, interesting :) The --sort-output was helpful to avoid unreadable merges between two language files, but actually... It should be removed since we don't care about the merges. The translation partner has powerful tools to check what is already translated, fuzzy, not translated with each iteration.

Thansk, we should change this in systematic also.